stable/artifactory/README.md

```diff
@@ -0,0 +1,116 @@
+# JFrog Artifactory Helm Chart
+
+## Prerequisites Details
+
+* Artifactory Pro trial license [get one from here](https://www.jfrog.com/artifactory/free-trial/)
+
+## Chart Details
+This chart will do the following:
+
+* Deploy Artifactory-oss
+* Deploy Artifactory-Pro
+
+## Installing the Chart
+
+To install the chart with the release name `artifactory`:
+
+```bash
+$ helm install --name artifactory stable/artifactory
+```
+
+### Deploying Artifactory OSS
+You can deploy Artifactory OSS by using the same instructions as provided for the Pro.
+**NOTE:** The Artifactory OSS does not use Nginx, so your Artifactory's Kubernetes service exposes Tomcat's port 8081 as port 80.
+
+By default it will run Artifactory-pro to run Artifactory-oss comment nginx configuration in value.yaml and use following command:
+Remove `nginx-deployment.yaml`, `nginx-pvc.yaml` and `nginx-service.yaml` and change art_service.externalPort to 80 in [values.yaml](values.yaml) before running command to install Artifactory-oss.
```

Comment:
This is not ideal. You cannot just remove something when you install the chart from the repo. The user would have to download and adapt it. Please add some conditional logic to not include what's not needed for OSS. BTW, why wouldn't you run the OSS version behind Nginx?